run/john.conf

```diff
@@ -395,6 +395,13 @@ AbortTemperature = 95
 
 
 # ZTEX specific settings
+[List.ZTEX:Devices]
+# If you list Serial Numbers (SN) of ZTEX boards here, it will display
+# numbers (starting from 1) instead of factory programmed SN's.
+# These numbers can be used in --dev command-line option.
+#04A31E0123
+#00AD034E01
```

Comment:
Should this possibly use the same example SNs that README-ZTEX does?

Reply:
I'm not sure if that matters. In both places there're example SNs rather than ones from actual boards.
Probably I can choose a few SNs that looks like they are "example" SNs, use them across the documentation.

Reply:
Right, that's what I suggest - choose some example SNs and use them consistently. Since you already had some in README-ZTEX, I thought you'd just continue to use them in other examples.